entry.js

```diff
@@ -2,9 +2,29 @@ import { jsx as _jsx } from 'react/jsx-runtime';
 import ReactDOM from 'react-dom';
 import { createElement } from 'react';
 
-export const App = ({ page }) => {
-  return createElement(page?.component, page?.props);
+import i18n from 'i18next';
+import { I18nextProvider } from 'react-i18next';
+
+export const App = ({ page, translations, locale }) => {
+  const i18nOptions = {
+    fallbackLng: 'en',
+    lng: locale,
+    resources: {
+      [locale]: translations,
+    },
+    interpolation: {
+      escapeValue: false // react already safes from xss
```

Comment:
*react already safe from xss

Aside: I'm hoping we can somehow use the entry.js from yss instead of duplicating it here. But that is unrelated to your change.